### DISCUSSION TITLE
Active Backends Value in CSV output?

### DISCUSSION THREAD

**Discussion by billglennon** [rapidloop/pgmetrics#46] [Q&A] [answered]
Hi,
I see the value Active Backends field and value in the output format = human. However, I do not see this field and value when the format = csv? Does this not exist in the csv output?
Thanks!

**Comment by mdevan** [comment, marked as answer]
In the CSV format, this is the key `pgmetrics.backends.count`.

**Reply by billglennon** [reply]
Thanks!
